The-SMIRNOFF-force-field-format.md

```diff
@@ -235,13 +235,23 @@ The sum of formal charges for the molecule or fragment will be used to determine
 * The `partial_charge_method` attribute (default: `"CM2"`) is used to specify how uncorrected partial charges are to be generated from the quantum chemical wavefunction. Later additions will add restrained electrostatic potential fitting (RESP) capabilities.   
 
 The `<ChargeIncrement>` tags specify how the quantum chemical derived charges are to be corrected to produce the final charges.
-The `charge#increment` attribute specify how much the charge on the associated tagged atom index (replacing `#`) should be modified.
+The `chargeincrement#` attribute specify how much the charge on the associated tagged atom index (replacing `#`) should be modified.
 The sum of charge increments should equal zero.
 
 Note that atoms for which library charges have already been applied are excluded from charging via `<ChargeIncrementModel>`.
 
 Future additions will provide options for intelligently fragmenting large molecules and biopolymers, as well as a `capping` attribute to specify how fragments with dangling bonds are to be capped to allow these groups to be charged.
 
+### `<ToolkitAM1BCC>`: Temporary support for basic charge calculation
```

Comment:
Suggest "Temporary support for toolkit-based AM1-BCC partial charges"

Reply:
Good call. Changed in d986ddf